src/EnergyPlus/EarthTube.cc

```diff
@@ -134,42 +115,27 @@ namespace EarthTube {
 	// Functions
 
 	void
+	clear_state()
+	{
+		TotEarthTube = 0;
+		EarthTubeSys.deallocate();
+		ZnRptET.deallocate();
+	}
```

Comment:
Just so you know, all those deallocates at the end of the units tests should go here in clear_state. Then there would be no need to deallocate them in unit tests.